gpcontrib/arenadata_toolkit/sql/upgrade_test.sql

```diff
@@ -80,14 +80,12 @@ BEGIN
 
 	IF 4 = (SELECT count(1)
```

Comment:
Can we use `JOIN` instead of `LEFT JOIN` + check `a.attrelid IS NOT NULL`, like:
```
	SELECT count(1)
			FROM pg_class c
			JOIN pg_attribute a ON a.attrelid = c.oid AND
			                       a.attname = 'tablespace_location'
			WHERE c.oid IN ('arenadata_toolkit.db_files_current'::regclass,
			                'arenadata_toolkit.__db_files_current'::regclass,
			                'arenadata_toolkit.__db_files_current_unmapped'::regclass,
			                'arenadata_toolkit.db_files_history'::regclass))
```

Reply:
done.